### PR TITLE
feat: add Dezh to partner and featured validator

### PR DESCRIPTION
**Pull request by ZigBalthazar**

### CODE REVIEW

**Review by b00f — APPROVED** [review]

File: website/assets/images/validators/dezh.png
Comment (Mr-HoDL58) [inline website/assets/images/validators/dezh.png:1]: This is not on a white background. All logo assets are white

File: website/index.html
Comment (Mr-HoDL58) [inline website/index.html:1]: needs to be white

File: website/assets/images/validators/dezh.png
Comment (Mr-HoDL58) [reply website/assets/images/validators/dezh.png:1]: You will find the template for PSD in assets

File: website/index.html
Comment (ZigBalthazar) [reply website/index.html:1]: we use template

File: website/assets/images/validators/dezh.png
Comment (ZigBalthazar) [reply website/assets/images/validators/dezh.png:1]: @Mr-HoDL58 
we use the template in `validators` directory 
the background color code is `#eae7ef`

**Pull request merged**